spec/helpers/report_helper_spec.rb

```diff
@@ -1,17 +1,29 @@
-require 'spec_helper'
+require "spec_helper"
 
-describe ReportHelper, :type => :helper do
+describe ReportHelper, type: :helper do
   before do
-    @comment = FactoryGirl.create(:comment)
-    @post = @comment.post
+    @user = bob
+    @post = @user.post(:status_message, text: "hello", to: @user.aspects.first.id)
+    @comment = @user.comment!(@post, "welcome")
+
+    @post_report = @user.reports.create({
+      :item_id => @post.id,
+      :item_type => 'post',
+      :text => 'offensive content'
+    })
+    @comment_report = @user.reports.create({
+      :item_id => @comment.id,
+      :item_type => 'comment',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.